pos_stock_available_online/models/res_config_settings.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    pos_display_product_quantity = fields.Boolean(
```

Comment:
res.configs.settings are per company

this fields should be on pos.config form view instead

Reply:
starting with Odoo 16 all settings specific to POS are defined in this model, you can check the original Odoo code. https://github.com/odoo/odoo/blob/b0df10c1e497f65b891f7141055e7f339d11b4b1/addons/point_of_sale/models/res_config_settings.py#L48-L105

Reply:
awesome, I didn't know that 

thanks!